src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -193,7 +193,7 @@ private StatementSyntax Transform( StatementSyntax stmt )
 		var expr = returnStmt.Expression;
 		if( m_generatedFunctionReturnsVoid && expr is not null ) {
 			if( IsStatementCompatibleExpression( expr ) ) {
-				return SyntaxFactory.ParseStatement( $"{Transform(expr).WithTriviaFrom(returnStmt)};\nreturn;" );
+				return SyntaxFactory.Block( SyntaxFactory.ExpressionStatement( Transform( expr ) ), SyntaxFactory.ReturnStatement() ).NormalizeWhitespace();

```

Comment:
`NormalizeWhitespace` iiis, not performant. Probably fine on small statements, but from previous experience (in static-di) it can really tank. I wouldn't want to use it here. It also won't end up being consistent with the surrounding code anyway.

I don't think we necessarily need to get the whitespace / style _perfect_, but just want to be thoughtful / intentional.

Reply:
Removed NormalizeWhitespace and replace with new formatting that looks like this:
```
[Blocking]
void IBar.Delete( Guid id ) {
	{ m_baz.Delete( id ); return; }
}
```